runtime/js/jslib.js

```diff
@@ -86,6 +86,7 @@ var caml_callback = caml_call_gen;
 //If: !doubletranslate
 //Requires: caml_stack_depth, caml_call_gen, caml_wrap_exception
 //Requires: caml_current_stack
+//Alias: caml_cps_trampoline
```

Comment:
I think this make the CI fail because wasmoo will "resolve" `caml_cps_trampoline` to `caml_callback`